### DISCUSSION TITLE
How to install the R package ospsuite

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#906] [Questions & Problems] [answered]
Hi, 

I tried to install the R package of the OSP suite, the version 9.1.1. The installation in R was successful, but when loading the package there was an error mentioning that the package rClr was missing. Is it normal, or something went wrong in my installation? Moreover, the package rClr is not available on CRAN, should I use the GitHub version?

Thanks a lot for your help!

Donato

**Comment by StephanSchaller** [comment]
Hi @teutonicod , yes, you NEED to use the GitHub Version for rClr :-)

**Comment by msevestre** [comment, marked as answer]
@teutonicod 
Please see the step by step installation guide in the repository
https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation

**Reply by teutonicod** [reply]
Hi guys, thanks a lot!

**Reply by teutonicod** [reply]
Sorry, still having issues. I found all the packages and the instructions, but the package ospsuite requires an R version up to the 3.6.3, while the R package rClr was built with the version 4.0.4. 
I installed everything on the version 3.6.3 but the package rClr gives an error since it was built with a newer version of R. And I could not find an older version of this package on the repository. Am I missing something?

**Reply by PavelBal** [reply]
Hi Donato,

for ospsuite 9.1.1 running with R3 please install this rClr release:
https://github.com/Open-Systems-Pharmacology/rClr/releases/tag/v0.8.3_Linux

and this ospsuite-r release:
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases/tag/v9.1.4

**Reply by teutonicod** [reply]
Thanks @PavelBal, _ already made some tests and now_ it seems to work corectly!

**Reply by msevestre** [reply]
In fact, we have updated the description to explain the difference between R3 and R4
Check it out

https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation
